packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -18,20 +18,21 @@ A test runner is provided for local development and should be invoked using the
-| spec                           | array   | false    |                         | A list of spec file paths.                                                                    |
+| Option Name                    | Alias | Type    | Required | Default                 | Description                                                                                   |
+|--------------------------------|-------|---------|----------|-------------------------|-----------------------------------------------------------------------------------------------|
+| browsers                       |       | array   | false    |                         | A list of browsers for the test run.                                                          |
+| config                         | c     | string  | false    |                         | A file path to the test runner configuration.                                                 |
+| formFactors                    |       | array   | false    |                         | A list of form factors for the test run. One of tiny, small, medium, large, huge, or enormous |
+| gridUrl                        |       | string  | false    |                         | The remote selenium grid address.                                                             |
+| keepAliveSeleniumDockerService |       | boolean | false    | false                   | Determines to keep the selenium docker service running upon test completion.                  |
+| locales                        |       | array   | false    | ['en']                  | A list of language locales for the test run.                                                  |
+| themes                         |       | array   | false    | ['terra-default-theme'] | A list of themes for the test run.                                                            |
+| hostname                       |       | string  | false    | localhost               | Automation driver host address.                                                               |
+| port                           |       | number  | false    | 4444                    | Automation driver port.                                                                       |
+| baseUrl                        |       | string  | false    |                         | The base url.                                                                                 |
+| suite                          |       | array   | false    |                         | Overrides specs and runs only the defined suites.                                             |
+| spec                           |       | array   | false    |                         | A list of spec file paths.                                                                    |
+| updateScreenshots              | u     | boolean | false    | false                   | Updates all reference screenshots with the latest screenshots.                                |
```

Comment:
After our own confusion these might be better to list in a single column with the flags:

| Option       | Type   | Default | Description                                   |
|--------------|--------|---------|-----------------------------------------------|
| --config, -c | string |         | A file path to the test runner configuration. |

I removed the `Required` column in this example, I'm not sure how much value it's providing since no options are required

Reply:
Added the aliases to the `Options` column and removed the `Alias` and `Required` columns. https://github.com/cerner/terra-toolkit/pull/568/commits/aff1cf1ec2f36c43efc3b6c8fe31743a4092fa01